### DISCUSSION TITLE
Only show the editor window?

### DISCUSSION THREAD

**Discussion by kjellaso** [eclipse-theia/theia#14302] [Q&A]
Hi! 

Would it possible to only show the editor window (hiding everything else) by for example passing a parameter to the application? Any tips on how i would go about to implement this?

Andreas

**Comment by msujew** [comment]
@tsmaeder Any clue? I believe the second window handling is doing something like that, right?

**Reply by tsmaeder** [reply]
No, secondary windows have a host of problems you don't want just for maximizing the editor. @kjellaso what is your goal in hiding the rest of the UI?

**Reply by kjellaso** [reply]
Would be to have a clean and simple editor view in one state that is embeded in a page, and then be able to go into the more detailed complex view in a fullscreen mode

**Reply by tsmaeder** [reply]
I guess a good place to start looking would be the `ApplicationShell` class. It's where the application window layout is defined.
